modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -236,5 +252,15 @@ function isFeatureIdDefined(value) {
   return value !== undefined && value !== null && value !== '';
 }
 
+function transformTileCoordsToWGS84(info, object, tile) {
```

Comment:
You need to check if the coords are already WGS84. MVTLayer does not always load features in tile coordinates.

Reply:
With `context.viewport.resolution` right?